backend/Application/Api/Rest/BalanceStatisticsController.cs

```diff
@@ -46,7 +46,7 @@ public async Task<ActionResult> GetLatest(string field, string? unit = "microccd
         {
             retValue = unit.ToLowerInvariant() switch
             {
-                "ccd" => retValue = ((decimal)value / 10_00_000).ToString(CultureInfo.InvariantCulture),
+                "ccd" => retValue = Math.Floor((decimal)value / 10_00_000).ToString(CultureInfo.InvariantCulture),
```

Comment:
So the idea is to not show any decimals? Usually when we show amounts in CCD we show them as `123.456`.

Can we at least document why this choice here is being made?

Reply:
@abizjak Sure, 
@mh-concordium asked us to make this change, probably because this is needed for integration into other exchanges.

Reply:
@mh-concordium : Kindly elaborate further

Reply:
@parv0888 could you please link to what issue is this fixing?

Reply:
I don't remember I made such a feedback at any point and issue 66 was created by you @parv0888 .  Looking at issues, I did create issue where I suggested showing all the decimals and aligning number to the right.

Reply:
@mh-concordium 
This is the issue that was communicated to me by @vj, We had a call about this.
I Created this issue so that its easier to document this on the PR. 
Im specifically unaware of the real reason why this was needed. So wanting your help in communicating this to @abizjak 
@coolmagic : Can you help?